ReactWindows/ReactNative.sln

```diff
@@ -11,22 +11,31 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ReactNative.Tests", "ReactN
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ChakraBridge", "ChakraBridge\ChakraBridge.vcxproj", "{4B72C796-16D5-4E3A-81C0-3E36F531E578}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ReactNative.Bridge", "ReactNative.Bridge\ReactNative.Bridge.csproj", "{BB680B12-67DE-4CC9-9AF4-B3622A02AA47}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ReactNative.Common", "ReactNative.Common\ReactNative.Common.csproj", "{1396E697-BE10-4A16-8B7E-C6AA7E826452}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
 		Debug|ARM = Debug|ARM
```

Comment:
I was futzing with settings when I was trying to get the new PCL project to build via msbuild. I can remove this now that I know we have to build the solution via devenv due to msbuild deficiencies.